### ISSUE TITLE
There is an issue with the DSC in the Abdomen dataset.

### ISSUE THREAD

**Issue by Zhenyu-040314** [Runshi-Zhang/UTSRMorph#5]
Thank you so much for your work!
When I reproduced your code on the Abdomen dataset, I noticed that the validation DSC initially increased and then decreased, while the loss function continued to decrease normally. I only changed the dataset path in your code and didn’t modify anything else. Have you encountered this issue before? My environment versions are:
pytorch=1.13.1
torchvision=0.14.1
pytorch-cuda=11.7
python=3.8.20
The UTSRMorph version for the OASIS dataset runs without any problems.
I am confusing about that. Looking forward to your reply！

![Image](https://github.com/user-attachments/assets/a36c7438-a5c4-4f64-89f0-969ebe65082e)

**Comment by Runshi-Zhang** [comment]
Yes, the issue is likely caused by overfitting due to the limited size of the dataset.

**Comment by Zhenyu-040314** [comment]
Thank you for your replying！
Due to overfitting issues caused by the Abdomen dataset, could you please explain how you selected the model corresponding to the best DSC in your table? We would like to adopt the same model selection method to ensure a fair comparison with the methods reported in our paper.
Looking forward to your reply！

![Image](https://github.com/user-attachments/assets/d9088c2d-363f-4781-a6c9-44a58f3370e9)

**Comment by Runshi-Zhang** [comment]
Thank you for your interest. Similar to other methods, we performed evaluation after each epoch and selected the model with the optimal Dice score. However, please note that during Dice calculation, two samples in the test set (img0002_tcia_CT and img0004_tcia_CT) were annotated with only three organs. Therefore, when computing the Dice for these two samples, the value (=0) corresponding to the missing organ should be excluded.

<img width="176" alt="Image" src="https://github.com/user-attachments/assets/4fe24431-210d-469e-bd87-75971686e28f" />

**Comment by Zhenyu-040314** [comment]
Thanks for the fast response!
I will perform the calculation according to your method.
Thank you very much!
